Library/OptimizelyClientType.swift

```diff
@@ -115,16 +113,8 @@ public func optimizelyProperties(environment: Environment? = AppEnvironment.curr
   }
 
   let allExperiments = optimizelyClient.allExperiments().map { experimentKey -> [String: String] in
-    let variation = try? optimizelyClient.getVariationKey(
```

Comment:
Question: Why did this change?

Reply:
A couple of relevant pieces of context:
- in phase 8 we are essentially deprecating the entirety of `optimizelyProperties`
- insights wants the format to be `session_variants_optimizely` : `[experimentKey1: variation, experimentKey2: variation ...]`

I decided to delete some of these properties here because after making the change to the format of `session_variants_optimizely` i realized those other values were not being used for anything.